### PR TITLE
Add systems tab to remediations detail

### PR DESCRIPTION
**Pull request by karelhala**
JIRA: https://issues.redhat.com/browse/RHCLOUD-13786

Screenshot from 2021-04-21 15-58-43
Screenshot from 2021-04-21 15-58-53
Screenshot from 2021-04-21 15-59-01

### CODE REVIEW

File: src/store/reducers.js [hunk 1]
```diff
@@ -26,6 +27,79 @@ function computeRebootStats(remediation) {
...
+                displayName={display_name}
+              />
+            ),
+            props: { width: 15 },
+          },
+          {
+            key: 'rebootRequired',
+            title: 'Reboot required',
+            // eslint-disable-next-line react/display-name
+            renderFunc: (rebootRequired) => (
```
Comment (Hyperkid123) [inline src/store/reducers.js:87]: Can we not store react components in redux?

File: src/store/reducers.js [hunk 2]
```diff
@@ -26,6 +27,79 @@ function computeRebootStats(remediation) {
   };
 }
 
+export const remediationSystems = ({ LOAD_ENTITIES_FULFILLED }) =>
+  applyReducerHash({
+    [ACTION_TYPES.SELECT_ENTITY]: (state, { payload }) => {
+      const selected = state.selected || new Map();
```
Comment (Hyperkid123) [inline src/store/reducers.js:33]: Maps are mutable and can cause issues within redux. Can we use a plain object instead?

File: src/store/reducers.js [hunk 3]
```diff
@@ -26,6 +27,79 @@ function computeRebootStats(remediation) {
...
+        } else if (payload.id === -1) {
+          selected.clear();
+        } else {
+          selected.delete(payload.id);
+        }
+      }
+
+      return {
+        ...state,
+        selected: new Map(selected),
```
Comment (Hyperkid123) [inline src/store/reducers.js:55]: ^^

File: src/store/reducers.js (hunk 2 above)
Comment (karelhala) [reply src/store/reducers.js:33]: The map is specifically used so we don't have to run `filters` and `finds` and removing the columns. I'll mark this as `TODO: refactor` and we can fix it later on.

File: src/store/reducers.js (hunk 1 above)
Comment (karelhala) [reply src/store/reducers.js:87]: I'll try looking at `cellTransformer` so we don't have to use the `renderFunc`

Comment (karelhala) [reply src/store/reducers.js:87]: Mhh `cellFormatters` wouldn't help us either as that's the same as `renderFunc` and inventory expects columns to be defined in redux and not trough props.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**